src/login/logind-action.c

```diff
@@ -366,6 +366,7 @@ static const char* const handle_action_verb_table[_HANDLE_ACTION_MAX] = {
         [HANDLE_SLEEP]                  = "sleep",
         [HANDLE_FACTORY_RESET]          = "perform a factory reset",
         [HANDLE_LOCK]                   = "be locked",
+        [HANDLE_SECURE_ATTENTION_KEY]   = "secure attention key pressed",
```

Comment:
I'm re-looking at this. Do I actually need this here?, along with changing the comment? I don't think HANDLE_SECURE_ATTENTION_KEY actually gets used now that I think about it.